### PR TITLE
feat: dark mode

### PR DESCRIPTION
**Pull request by allancorbett**
BREAKING CHANGE: it could all go horribly wrong...

See https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme for more info. 

Safari and FF67 support it at the moment with Chrome going to support it. 

Screenshot 2019-04-10 at 10 15 42

Screenshot 2019-04-10 at 10 15 56

**Things still needing done:**
- [x] "What we do" images
- [ ] General discussion about if we want to support this or not...

### CODE REVIEW

**Comment by james-nash** [comment]
Thanks for proposing & implementing this feature!

I think it's a good addition. It's something we were thinking of possibly doing in `next` via the new color system (see #147), but we never really thought about doing it for the _current_ 1.x version. However, why not? Especially since you've pretty much done it already.

So, I propose:
1. Polish this off and let's get it released
1. We'll update the current Buildit website to use this (and figure out what to do about those "What we do" images* in the process
1. In `next`, once the new color system goes in, we'll retrofit an equivalent dark mode (should be pretty easy to implement)

@danbull @dw-buildit : What say ye?


*) Off the top of my head we could:
* Inline those SVGs and set their fill to `currentColor` so they automagically adapt
* Produce white versions of them and use a `<picture>` element to select the light or dark image based on the `prefers-color-scheme` media query.

**Comment by allancorbett** [comment]
> Inline those SVGs and set their fill to currentColor so they automagically adapt

That would be my preference.

**Comment by allancorbett** [comment]
So I've implented the dark mode for certain images using some (probably rather ropey) methods: `src/ui-lib/sass/05-components/01-atoms/04-media/_24-image.scss`

Thoughts?

File: package.json [hunk 1]
```diff
@@ -43,16 +43,12 @@
     "url": "http://buildit.wiprodigital.com/"
   },
   "contributors": [
-    "Gregg Anderson",
```
Comment (james-nash) [inline package.json]: Not sure why this is showing up as a change versus `develop`. We need to be careful when this PR eventually gets merged that it doesn't revert the recent update to the contributors list.

File: src/ui-lib/sass/05-components/01-atoms/04-media/_24-image.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.grav-c-image {
+  &__invertible {
+    @media (prefers-color-scheme: dark) {
+      filter: invert(100);
```
Comment (james-nash) [inline src/ui-lib/sass/05-components/01-atoms/04-media/_24-image.scss]: I think this is fine. Especially since it's opt-in via the modifier class.

**Comment by james-nash** [comment]
Looks good to me - just watch out with the `package.json` because it looks like it might overwrite the recent updates to that.

File: package.json (hunk 1 above)
Comment (allancorbett) [reply package.json]: Not sure - fixed in another commit though.

File: src/ui-lib/sass/05-components/01-atoms/04-media/_24-image.scss (hunk 2 above)
Comment (allancorbett) [reply src/ui-lib/sass/05-components/01-atoms/04-media/_24-image.scss]: 👍

**Comment by james-nash** [comment]
This is lookin' good to me.

I'm happy for this PR to be un-drafted (is that the right term?) and merged.

I see you've already prepped some associated changes on the website, so let's get this released as soon as we can. :-)

**Pull request merged**